packages/iov-faucets/tslint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../tslint.json"
```

Comment:
I added these mostly for uniformity with the other packages. I don't think it's strictly necessary, although possibly IDEs might get confused sometimes if you've opened a package rather than the whole repo?

Reply:
okay, feel free to keep it to make explicit what we want.

But opening one package in an IDE will probably not work since all the node_modules are not there.